cmake/sysbuild/suit.cmake

```diff
@@ -534,12 +534,18 @@ function(suit_setup_merge)
       list(APPEND ARTIFACTS_TO_MERGE ${BINARY_DIR}/zephyr/suit_mpi_${IMAGE_TARGET_NAME}_merged.hex)
     endif()
 
+    # Get a list of files (and their dependencies) which need merging into the uicr merged file and
+    # add them at the end of the list, allowing for overwriting
+    get_property(merge_files GLOBAL PROPERTY SUIT_MERGE_FILE)
+    get_property(merge_dependencies GLOBAL PROPERTY SUIT_MERGE_DEPENDENCIES)
+
     set_property(
       GLOBAL APPEND PROPERTY SUIT_POST_BUILD_COMMANDS
       COMMAND ${PYTHON_EXECUTABLE} ${ZEPHYR_BASE}/scripts/build/mergehex.py
       --overlap replace
       -o ${OUTPUT_HEX_FILE}
-       ${ARTIFACTS_TO_MERGE}
+       ${ARTIFACTS_TO_MERGE} ${merge_files}
```

Comment:
observation: this is really not self-explaining what the difference between `ARTIFACTS_TO_MERGE` and `merge_files` is.
But as this file in general is messy, then there is not change requested for this PR.